content/documentation/staging/features/index.adoc

```diff
@@ -529,6 +529,28 @@ Kiali uses Istio Wizards to generate Istio Traffic config for a specific Service
 </div>
 ++++
 
+== Health Configuration
+
+Kiali allows users to customize the configuration used to determine health.
```

Comment:
This deserves more wording.
Like the health is a composition of several inputs, pods, traffic, etc.
Global health is calculated like the max of all inputs and Health Configuration allows to determine the health of a particular signal.

On the status of the feature, traffic health is supported.

Re-arrange my wording, but I guess it can give you an idea about what I mean.